File: tests/struct/copy-and-edit.js
export function pointOffsetFromOrigin$x$y$(x, y) {
  let result = {
    x: origin.x,
    y: origin.y
  };
  result.x = result.x + x;
  result.y = result.y + y;
  return result;
}
let origin = {
  x: 0,
  y: 0
};

File: tests/struct/copy-and-edit.swift
public struct Point {
    var x: Double
    var y: Double
}

var origin = Point(x: 0, y: 0)

public func pointOffsetFromOrigin(x: Double, y: Double) -> Point {
	var result = origin
	result.x += x
	result.y += y
	return result
}
